source/docs/controls-overviews/control-system-hardware.rst

```diff
@@ -68,6 +68,14 @@ The [CTRE Power Distribution Panel 2.0](https://store.ctr-electronics.com/produc
 
 The [REV Power Distribution Hub](https://docs.revrobotics.com/ion-control-system/pdh/overview) (PDH) is designed to distribute power from a 12VDC battery to various robot components.  The PDH features 20 high-current (40A max) channels, 3 low-current (15A max), and 1 switchable low-current channel. The Power Distribution Hub features toolless latching WAGO terminals, an LED voltage display, and the ability to connect over CAN or USB-C to the REV Hardware Client for real-time telemetry.
 
+## AndyMark Power Distribution Board
+
+.. image:: images/control-system-hardware/andymark-ampd.jpg
+  :alt: AndyMark Power Distribution Board
+  :width: 500
+
+The [AndyMark Power Distribution Board](https://andymark.com/collections/new-featured-products/products/ampd-andymark-power-distribution) (AMPD) is designed to distribute power from a 12VDC battery to various robot components. The AMPD features 24 fully insulated 40A ports that work with ATO/ATC breakers. With toolless lever connectors and 22.5 degree angled outputs for clean wiring, the AMPD is designed to simplify robot wiring.
```

Comment:
This feels like a link that's less likely to change in the future: https://andymark.com/products/ampd-andymark-power-distribution